src/components/nav/nav.tsx

```diff
@@ -0,0 +1,201 @@
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Prop,
+  State,
+  VNode
+} from "@stencil/core";
+import { slotChangeHasAssignedElement } from "../../utils/dom";
+import { CSS, LEVEL, SLOTS } from "./resources";
+
+type Level = "primary" | "secondary" | "tertiary";
+
+@Component({
+  tag: "calcite-nav",
+  styleUrl: "nav.scss",
+  shadow: {
+    delegatesFocus: true
+  }
+})
+
+/**
+ * @slot logo - A slot for adding a `calcite-logo` component to the primary nav level.
+ * @slot user - A slot for adding a `calcite-user` component to the primary nav level.
+ * @slot progress - A slot for adding a `calcite-progress` component to the primary nav level.
+ * @slot menu-action - A slot for adding a `calcite-action` component to the primary nav level.
+ * @slot primary-content-start - A slot for adding a `calcite-menu`, `calcite-action`, or other interactive elements in the start position of the primary nav level.
+ * @slot primary-content-center - A slot for adding a `calcite-menu`, `calcite-action`, or other interactive elements in the center position of the primary nav level.
```

Comment:
Would "primary nav level", "secondary nav level", "tertiary..." be the same as "component's primary level", "component's secondary level", "component's tertiary..."?

Reply:
Agreed sounds good